main.cpp

```diff
@@ -179,6 +237,10 @@ void run()
   check_solution<T>(num_times, a, b, c, sum);
```

Comment:
Verification printing fixed in 5a64ce1.
#20 still needs to be addressed, although all this MPI code assumes `T` is `double` as I've used `MPI_Double` for messages. We should probably support `MPI_Float` too.